samples/event_manager_proxy/boards/nf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BOARD_ENABLE_CPUNET=y
```

Comment:
Filename is wrong. You are missing `r`.

Reply:
Right. There was second error before that hides this one during tests.